test/introspection-filter-basics.js

```diff
@@ -659,8 +659,7 @@ test('repeatable directive', async (t) => {
     filterSchema: true,
     authDirective: 'counter',
     applyPolicy: async () => {
-      t.pass('should be called once')
-      t.todo('should be called three times but repeatable directives are not supported')
+      t.pass('should be called three times')
```

Comment:
If we go with the serial implementation, this function must return `true` or it will be called once again